pkgs/development/python-modules/marisa/default.nix

```diff
@@ -7,12 +7,14 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "s-yata";
     repo  = "marisa-trie";
-    rev   = "59e410597981475bae94d9d9eb252c1d9790dc2f";
-    sha256 = "0z4bf55np08q3cbi6gvj3cpw3zp8kf2d0jq6k74pjk066m7rapbb";
+    rev   = "8dba9850b89d7828ebf33b8ab84df2b54d31260b";
+    sha256 = "0pkp9fggk53lxlicfwrskgx33qplc4v6njbavlnz4x4z63zd4933";
   };
 
   nativeBuildInputs = [ swig marisa ];
   buildinputs = [ marisa ];
```

Comment:
There is a typo in here: `buildInputs` and then the `NIX_CFLAGS_LINK` line should be no longer necessary.

Reply:
You're right, `NIX_CFLAGS_COMPILE` isn't necessary either. Strange it was working with the typo until recently.